### ISSUE TITLE
Add ability to search for a message in a specific chat

### ISSUE THREAD

**Issue by oorion** [cashweb/stamp#254]

**Comment by sysfu** [comment]
This would be nice and is more or less a standard feature on most messaging apps. See Delta Chat implementation if you need an idea. [get.delta.chat](get.delta.chat)

As a workaround I'm using the browser search to find messages is the meantime.

**Comment by schancel** [comment]
Yeah, there's a ton to be done here. I think delta.chat is a good model. They're doing something similar but just using SMTP. The protocol used by Stamp is similar to SMTP/POP but with encrypted messages and payments.

I'm happy to accept PRs, but there's so many bugs irritating me that I'm working on. I've taken a vow of Stamp so I am forced to fix what bothers me. :)
